.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+#### What does this PR do?
+
+#### How can this change be manually tested?
```

Comment:
Can we revert this file? I actually have another PR in the works to address this: https://github.com/jeremyckahn/farmhand/pull/125